### PR TITLE
Generate meta data as a plain object

### PR DESCRIPTION
**Pull request by msujew**
Removes unnecessarily created files and simplifies the `LanguageMetaData` setup.

### CODE REVIEW

File: packages/langium-cli/src/generator/module-generator.ts [hunk 1]
```diff
@@ -21,10 +21,16 @@ export function generateModule(grammar: langium.Grammar, config: LangiumConfig):
     node.append(
         'import { ', grammar.name, "AstReflection } from './ast';", NL,
         'import { ', grammar.name, "GrammarAccess } from './grammar-access';", NL,
-        'import { ', grammar.name, "LanguageMetaData } from './meta-data';", NL,
         "import { Parser } from './parser';", NL, NL
     );
 
+    node.append('const metaData = {', NL);
```
Comment (spoenemann) [inline packages/langium-cli/src/generator/module-generator.ts:27]: Shall we export this so you can access it directly in your language implementation?

File: packages/langium-cli/src/generator/module-generator.ts [hunk 2]
```diff
@@ -21,10 +21,16 @@ export function generateModule(grammar: langium.Grammar, config: LangiumConfig):
...
+    node.indent(metaData => {
+        metaData.append(`languageId: '${config.languageId}',`, NL);
```
Comment (spoenemann) [inline packages/langium-cli/src/generator/module-generator.ts:29]: What if this property is undefined? Shall we make it mandatory in the config?

File: packages/langium-cli/src/generator/module-generator.ts [hunk 3]
```diff
@@ -21,10 +21,16 @@ export function generateModule(grammar: langium.Grammar, config: LangiumConfig):
...
+        metaData.append(`fileExtensions: [${config.extensions && config.extensions.map(e => appendQuotesAndDot(e)).join(', ')}]`, NL);
```
Comment (spoenemann) [inline packages/langium-cli/src/generator/module-generator.ts:30]: Seeing this I think we could rename `extensions` to `fileExtensions` in the config as well, WDYT?

File: packages/langium-cli/src/generator/module-generator.ts (hunk 2 above)
Comment (msujew) [reply packages/langium-cli/src/generator/module-generator.ts:29]: Yes #158 already makes it mandatory.

File: packages/langium-cli/src/generator/module-generator.ts (hunk 3 above)
Comment (msujew) [reply packages/langium-cli/src/generator/module-generator.ts:30]: Yeah, that makes sense.

File: packages/generator-langium/langium-template/package.json [hunk 4]
```diff
@@ -29,7 +29,7 @@
     "langium": {
         "languageId": "<%= language-id %>",
         "grammar": "src/language-server/<%= language-id %>.langium",
-        "extensions": <%= file-extension %>,
+        "fileExtensions": <%= file-extension %>,
```
Comment (spoenemann) [inline packages/generator-langium/langium-template/package.json:32]: This needs an entry in the `CHANGELOG.md`. Could you create one in `langium-cli`?

You can take https://github.com/eclipse/lsp4j/blob/master/CHANGELOG.md as an example format.

File: examples/arithmetics/src/language-server/generated/module.ts [hunk 5]
```diff
@@ -6,14 +6,18 @@
 import { LangiumGeneratedServices, LangiumServices, Module } from 'langium';
 import { ArithmeticsAstReflection } from './ast';
 import { ArithmeticsGrammarAccess } from './grammar-access';
-import { ArithmeticsLanguageMetaData } from './meta-data';
 import { Parser } from './parser';
 
+const metaData = {
```
Comment (spoenemann) [inline examples/arithmetics/src/language-server/generated/module.ts:11]: This needs to be re-generated.

**Review by spoenemann — APPROVED** [review]

**Pull request merged**